autoendpoint/src/routers/apns/router.rs

```diff
@@ -218,14 +199,6 @@ impl Router for ApnsRouter {
             serde_json::to_value(app_id).unwrap(),
         );
 
-        if let Some(aps) = &router_input.aps {
```

Comment:
It doesn't look like aps was solely used for "testing" (at least unit testing). It comes in from `RouterDataInput` from new `registration`/`registration updates`. [firefox-ios may pass one in](https://github.com/mozilla-mobile/firefox-ios/blob/main/Push/PushClient.swift#L53) when its `experimentalMode` (is that dev mode?) is enabled.

It seems like it was ultimately added for dev use?
https://bugzilla.mozilla.org/show_bug.cgi?id=1364403
https://github.com/mozilla-services/autopush/issues/910#issuecomment-304953536

So we probably want to keep it or at least check w/ the ios team before removing it?